flocker/node/agents/test/test_blockdevice.py

```diff
@@ -4119,13 +4125,14 @@ class FakeProfiledLoopbackBlockDeviceIProfiledBlockDeviceTests(
     """
 
 
-def umount(device_file):
+def umount(unmount_target):
     """
     Unmount a filesystem.
 
-    :param FilePath device_file: The device file that is mounted.
+    :param FilePath unmount_target: The device file that is mounted or
```

Comment:
Looks like it's always the mountpoint now, but in it was both before this PR.

I'm sort of inclined to keep this comment as is with this PR, because we have used this code for either in the past, and we don't need to change the body of this function if we change the way we use it. This is also the interface that is exposed by the unmount command line utility and system call so it should be a fairly recognizable API.